### PR TITLE
DEVAB-455 adding white space check

### PR DESCRIPTION
**Pull request by danieldmo**
super simple PR

### CODE REVIEW

File: api/controllers/AuthController.js [hunk 1]
```diff
@@ -86,6 +86,12 @@ module.exports = {
       inviteToken : req.param('inviteToken')
     };
 
+    // checking password for white spaces and tabs
```
Comment (danieldmo) [inline api/controllers/AuthController.js:89]: only thing that was added in this PR

Comment (markmur) [inline api/controllers/AuthController.js:89]: `/\s/g` is the correct check for spaces in a regex

Comment (danieldmo) [inline api/controllers/AuthController.js:89]: what is the difference ?

Comment (markmur) [inline api/controllers/AuthController.js:89]: `\s` is the regex standard for whitespace. Not sure all browsers would treat `/ /` correctly

Comment (danieldmo) [inline api/controllers/AuthController.js:89]: I am pretty sure js interprets both the same way.

Comment (markmur) [inline api/controllers/AuthController.js:89]: Not sure why you'd want to deviate from the standard. `\s` covers both whitespace and tab

Comment (danieldmo) [inline api/controllers/AuthController.js:89]: so does / /

**Pull request merged**